.readthedocs.yaml

```diff
@@ -0,0 +1,35 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
```

Comment:
Is there a reason to not keep this consistent with the rest of Pontoon and use 3.10?

Reply:
I just took it from https://docs.readthedocs.io/en/stable/config-file/v2.html, assuming this is the latest version. 

Since docs environment is not running the main codebase and has different set of requirements, I wonder if we should keep the python versions of the two in sync?

Reply:
> Since docs environment is not running the main codebase and has different set of requirements, I wonder if we should keep the python versions of the two in sync?

My main concern is the fact that we have a generated list of requirements, and that might change depending on the Python version 🤔

Reply:
These requirements are not generated using `make requirements`. I was thinking of taking care of this as part of updating the deps, but that's a separate patch.

I'm changing python version to 3.10.